metaflow/plugins/kfp/nested_parallelfor.ipynb

```diff
@@ -25,20 +23,20 @@
     "\n",
     "def pipeline_transformer(op: ContainerOp):\n",
     "    op.execution_options.caching_strategy.max_cache_staleness = \"P0D\"\n",
-    "    op.container.set_cpu_request(\"100m\")\n",
-    "    op.container.set_cpu_limit(\"150m\")\n",
+    "    op.container.set_cpu_request(\"150m\")\n",
+    "    op.container.set_cpu_limit(\"150m\") # please ensure the limit and requests are equal\n",
```

Comment:
This is the only meaningful changes in this large diff (a reminder to customers to keep limits and requests the same).